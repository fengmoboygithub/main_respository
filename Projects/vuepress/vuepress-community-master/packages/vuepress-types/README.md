# vuepress-types

> VuePress TypeScript helpers

## Documentation

[vuepress.github.io](https://vuepress.github.io)

## License

[MIT](https://github.com/vuepress/vuepress-community/blob/master/LICENSE) &copy; [VuePress Community](https://github.com/vuepress)
