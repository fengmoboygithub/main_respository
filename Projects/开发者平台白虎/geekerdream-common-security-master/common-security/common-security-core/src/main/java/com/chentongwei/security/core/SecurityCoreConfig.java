package com.chentongwei.security.core;

import com.chentongwei.security.core.properties.SecurityProperties;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.PropertySource;

/**
 * 安全模块核心配置
 *
 * @author chentongwei@bshf360.com 2018-05-25 11:31
 */
@Configuration
@PropertySource("classpath:security.properties")
@EnableConfigurationProperties(SecurityProperties.class)
public class SecurityCoreConfig {

}
