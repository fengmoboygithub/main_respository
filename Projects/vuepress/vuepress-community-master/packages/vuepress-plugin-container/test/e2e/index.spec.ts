import { App } from 'vuepress-types'
import { fs, path, parseFrontmatter } from '@vuepress/shared-utils'
import { createApp } from '@vuepress/core'

const fragmentDir = path.join(__dirname, '__fragments__')

describe('vuepress-plugin-container', () => {
  let app: App

  beforeAll(async () => {
    app = createApp({
      temp: path.resolve(__dirname, '.temp'),
      plugins: [
        [
          'vuepress-plugin-container',
          {
            type: 'hint',
            defaultTitle: {
              '/': 'ðŸ’¡ HINT',
              '/jp/': 'ãƒ’ãƒ³ãƒˆ',
            },
          },
        ],
        [
          'vuepress-plugin-container',
          {
            type: 'theorem',
            before: (info: string): string =>
              `<div class="theorem"><p class="title">${info}</p>`,
            after: '</div>',
          },
        ],
      ],
    })
    return app.process()
  })

  fs.readdirSync(fragmentDir).forEach((name: string) => {
    const filePath = path.join(fragmentDir, name)
    const rawFile = fs.readFileSync(filePath, 'utf8')
    const { data, content } = parseFrontmatter(rawFile)
    test(name, () => {
      const { html } = app.markdown.render(content, {
        ...(data.ENV || {}),
        frontmatter: data,
      })
      expect(html).toMatchSnapshot()
    })
  })
})
