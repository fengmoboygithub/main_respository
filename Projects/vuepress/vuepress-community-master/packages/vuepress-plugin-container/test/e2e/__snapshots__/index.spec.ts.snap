// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`vuepress-plugin-container hint-default.md 1`] = `
<div class="custom-block hint">
  <p class="custom-block-title">💡 HINT</p>
  <p>This is a hint.</p>
</div>
`;

exports[`vuepress-plugin-container hint-locale-jp.md 1`] = `
<div class="custom-block hint">
  <p class="custom-block-title">ヒント</p>
  <p>これはヒントです。</p>
</div>
`;

exports[`vuepress-plugin-container hint-override.md 1`] = `
<div class="custom-block hint">
  <p class="custom-block-title">提示</p>
  <p>这是一句提示。</p>
</div>
`;

exports[`vuepress-plugin-container theorem.md 1`] = `
<div class="theorem">
  <p class="title">Newton's First Law</p>
  <p>In an inertial frame of reference, an object either remains at rest or continues to move at a constant velocity, unless acted upon by a force.</p>
</div>
`;
